app/[localeCode]/layout.tsx

```diff
@@ -18,7 +18,8 @@ import {
     BroadcastStoryProvider,
     BroadcastTranslationsProvider,
 } from '@/modules/Broadcast';
-import { CookieConsent } from '@/modules/CookieConsent';
+import { CookieConsentProvider } from '@/modules/CookieConsent';
+import { CookieConsent } from '@/modules/CookieConsent/CookieConsent';
```

Comment:
It has to be separate. If I export CookieConsent from module root, I get this error:
<img width="1014" alt="image" src="https://github.com/user-attachments/assets/00dfa813-f60d-47dd-a73e-e0312a9f8c69" />

which is caused by `await app().newsroom()` usage in `CookieConsent` component